templates/resources/index.html

```diff
@@ -0,0 +1,89 @@
+    </div>
+    {% else %}
+    <ul class='p-list'>
+      {% for item in case_studies %}
+        <li class="p-list__item"><a href="{{item.link}}">{{item.title.rendered | safe}}&nbsp;&rsaquo;</a></li>
+      {% endfor %}
+    </ul>
+    {% endif %}
+  </div>
+</section>
+
+{% get_json_feed "https://insights.ubuntu.com/wp-json/wp/v2/posts?categories=1187" limit=10 as webinars %}
+<section class="p-strip is-shallow">
+  <div class="row">
+    <h2>Webinars</h2>
+    {% if webinars is False %}
+    <div class="p-notification--negative">
+      <p class="p-notification__response"><span class="p-notification__status">Error:</span>The live news feed failed to load. Please <a href="https://github.com/canonical-websites/www.ubuntu.com/issues/new" class="external">report this bug</a> and our team will fix the problem as soon as&nbsp;possible.</p>
+    </div>
+    {% else %}
+    <ul class='p-list'>
+      {% for item in webinars %}
+        <li class="p-list__item"><a href="{{item.link}}">{{item.title.rendered | safe}}&nbsp;&rsaquo;</a></li>
+      {% endfor %}
+    </ul>
+    {% endif %}
+  </div>
+</section>
+
+{% get_json_feed "https://insights.ubuntu.com/wp-json/wp/v2/posts?categories=1485" limit=10 as white_papers %}
+<section class="p-strip is-shallow">
+  <div class="row">
+    <h2>White papers</h2>
+    {% if white_papers is False %}
+    <div class="p-notification--negative">
+      <p class="p-notification__response"><span class="p-notification__status">Error:</span>The live news feed failed to load. Please <a href="https://github.com/canonical-websites/www.ubuntu.com/issues/new" class="external">report this bug</a> and our team will fix the problem as soon as&nbsp;possible.</p>
+    </div>
+    {% else %}
+    <ul class='p-list'>
+      {% for item in white_papers %}
+        <li class="p-list__item"><a href="{{item.link}}">{{item.title.rendered | safe}}&nbsp;&rsaquo;</a></li>
+      {% endfor %}
+    </ul>
+    {% endif %}
+  </div>
+</section>
+
+{% get_json_feed "https://insights.ubuntu.com/wp-json/wp/v2/posts?categories=2497" limit=10 as tutorials %}
+<section class="p-strip is-shallow">
+  <div class="row">
+    <h2>Tutorials</h2>
+    {% if tutorials is False %}
+    <div class="p-notification--negative">
+      <p class="p-notification__response"><span class="p-notification__status">Error:</span>The live news feed failed to load. Please <a href="https://github.com/canonical-websites/www.ubuntu.com/issues/new" class="external">report this bug</a> and our team will fix the problem as soon as&nbsp;possible.</p>
+    </div>
+    {% else %}
+    <ul class='p-list'>
+      {% for item in tutorials %}
+        <li class="p-list__item"><a href="{{item.link}}">{{item.title.rendered | safe}}&nbsp;&rsaquo;</a></li>
```

Comment:
Do we need something for if there are no results?

Reply:
Good idea